### PR TITLE
fix: add comparison version

### PR DESCRIPTION
**Pull request by fajarhambali**

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=ait-spring-boot-template&pullRequest=6) (https://sonar.akarinti.tech/dashboard?id=ait-spring-boot-template&pullRequest=6)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=6)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=6)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=6)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=ait-spring-boot-template&pullRequest=6&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&metric=new_coverage&view=list&pullRequest=6)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&metric=new_duplicated_lines_density&view=list&pullRequest=6)

File: src/main/java/org/ait/project/guideline/example/modules/appversion/service/adapter/query/impl/AppVersionAdapterImpl.java
Comment (bintang-dev) [inline src/main/java/org/ait/project/guideline/example/modules/appversion/service/adapter/query/impl/AppVersionAdapterImpl.java:1]: Nama class diharapkan ada prefix Query contoh
AppVersionQueryAdapter ini untuk menunjukkan adapter digunakan melakukan query, jadi tidak ambigu jika ada adapter command

**Review by bintang-dev — CHANGES_REQUESTED** [review]
ada revisi di penamaan query adapter

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=ait-spring-boot-template&pullRequest=6) (https://sonar.akarinti.tech/dashboard?id=ait-spring-boot-template&pullRequest=6)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=6)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=6)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=6)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=ait-spring-boot-template&pullRequest=6&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage:  [0.00% Coverage](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&metric=new_coverage&view=list&pullRequest=6)

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=ait-spring-boot-template&metric=new_duplicated_lines_density&view=list&pullRequest=6)

**Pull request merged**